app/api/custom/events.py

```diff
@@ -19,3 +24,33 @@ def get_dates(event_id):
         ))[0]))
     )
     return jsonify(dates)
+
+@events_routes.route('/<int:event_id>/contact-organizer', methods=['POST'])
+@jwt_required
+def resend_invite(event_id):
```

Comment:
Why resent_invite?

Reply:
> Why resent_invite?


Ohh by mistake sir. Got some errors while commiting. While solving these errors, i done this my mistake.